pallets/msa/src/weights.rs

```diff
@@ -18,7 +18,7 @@
 //! Autogenerated weights for pallet_msa
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2023-01-27, STEPS: `20`, REPEAT: 10, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2023-02-09, STEPS: `20`, REPEAT: 10, LOW RANGE: `[]`, HIGH RANGE: `[]`
 //! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("frequency-bench"), DB CACHE: 1024
```

Comment:
Can you please just checking the weights for MSA?

Reply:
I only checked in the MSA weights because two new benchmarks were added to it.  Jenkins created the rest.  I will work with @shannonwells to remove them.